build.gradle

```diff
@@ -77,8 +77,8 @@ dependencies {
     compile("org.springframework.security:spring-security-jwt:1.0.10.RELEASE")
 
     compile("net.logstash.logback:logstash-logback-encoder:5.2")
-    implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.4.0-BETA'
-    implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:1.2.0-BETA'
+    compile("com.microsoft.azure:applicationinsights-logging-logback:2.4.0-BETA")
```

Comment:
latest gradle syntax is to use `implementation`, not `compile`.  Don't need the brackets since it is groovy and single quotes preferred when you don't need to do interpolation - i.e. revert this change :grin:

Reply:
I was eliminating differences from other services, but its not this, so happy to re-instate.